traffic-policy/actions/forward-internal/behavior.mdx

```diff
@@ -0,0 +1,23 @@
+## Behavior
+
+When the Forward Internal action executes, it will look up the specified endpoint and relay any incoming traffic. The endpoint being forwarded to will behave as if the traffic was sent directly to it and execute all of the actions in its associated Traffic Policy. The endpoint being forwarded to must exist in the same ngrok account and be the same protocol as the forwarding endpoint (e.g. an HTTP public endpoint can only forward to an HTTP private endpoint). You also may not forward traffic to a second private endpoint or back to the original endpoint in a loop.
+
+If the forwarding is successful, any response will be returned to the forwarding endpoint, no further actions in the Traffic Policy will be executed, and no traffic will be sent to the upstream.
+
+If the forwarding is unsuccessful because the specified endpoint doesn't exist, is offline, or encounters another error the action will return an error and follow the behavior that is specified by `on_error` (see [Managing Fallback Behavior](#on-error)).
```

Comment:
```suggestion
If the forwarding is unsuccessful because the specified endpoint doesn't exist, is offline, or encounters another error, the action will return an error and follow the behavior that is specified by `on_error` (see [Managing Fallback Behavior](#on-error)).
```

Reply:
I swear I had that comma in there at some point lmao